eclipse-scout-core/src/tabbox/SimpleTab.less

```diff
@@ -81,6 +81,43 @@
       &.closer {
         margin-left: 7px;
       }
+
+      &.icon {
+        margin-right: 5px;
+        vertical-align: middle;
+
+        &.font-icon {
+          font-size: 14px;
+        }
+      }
+
+      &.message {
+        margin-right: 5px;
+        vertical-align: middle;
+        font-size: 14px;
+      }
+
+      &.notification-badge {
+        background: @notification-badge-background-color;
+        color: @notification-badge-color;
+        top: 2px;
+        padding: 1px 6px;
+        border-radius: 8px;
+        font-size: @font-size-small;
+        font-weight: @font-weight-bold;
+        float: right;
```

Comment:
I think the order of the status icons should be preserved to give control over where they should be placed. With float: right this is not be possible anymore.

Reply:
Done 👍🏼